templates/ntpd.sysconfig.j2

```diff
@@ -1,3 +1,3 @@
 # {{ ansible_managed }}
 
-OPTIONS="-g{{ ' -u ntp:ntp -p /var/run/ntpd.pid' if ansible_distribution in ['RedHat', 'CentOS'] and ansible_distribution_major_version|int < 7 else '' }}"
+OPTIONS="-g{{ ' -u ' + timesync_ntp_user + ':' + timesync_ntp_user + ' -p /var/run/ntpd.pid' if ansible_distribution in ['RedHat', 'CentOS'] and ansible_distribution_major_version|int < 7 else '' }}"
```

Comment:
Do we need `timesync_ntp_user + ':' + timesync_ntp_group` here or is it always guaranteed that `timesync_ntp_user == timesync_ntp_group`

Reply:
I believe this is the basic configuration, that doesn't typically make sense to change; but, adding a var as default.